lib/oneview-sdk/resource/api1800/c7000/logical_interconnect.rb

```diff
@@ -0,0 +1,42 @@
+# (C) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+require_relative '../../api1600/c7000/logical_interconnect'
+
+module OneviewSDK
+  module API1800
+    module C7000
+      # Logical interconnect resource implementation for API1800 C7000
+      class LogicalInterconnect < OneviewSDK::API1600::C7000::LogicalInterconnect
+        # Gets igmpSettings of logical interconnect
+        def get_igmp_settings
+          ensure_client && ensure_uri
+          response = @client.rest_get("#{@data['uri']}/igmpSettings")
```

Comment:
why "#" before {@data['uri']}

Reply:
This is just to interpret whatever given inside double quotes which is basically equals to rest_get(@data['uri'] + '/igmpSettings')

Reply:
ok